ci/build-push-multiplatform.sh

```diff
@@ -2,21 +2,28 @@
 
 set -eo pipefail
 
-while ! docker buildx ls; do
-    echo "Cannot connect to docker daemon"
-    sleep 1
-done
-
-DOCKER_BUILDX_LS_OUT=$(docker buildx ls <<-END
+if echo "${PLATFORM}" | grep windows; then
+    while ! docker images; do
+        echo "Cannot connect to docker daemon"
+        sleep 1
+    done
+else
+    while ! docker buildx ls; do
+        echo "Cannot connect to docker daemon"
+        sleep 1
+    done
+
+    DOCKER_BUILDX_LS_OUT=$(docker buildx ls <<-END
```

Comment:
Why do we need to do this?

Reply:
To verify if we have connection with docker server